mmdet/models/utils/transformer.py

```diff
@@ -1,16 +1,28 @@
+import math
+
 import torch
 import torch.nn as nn
 from mmcv.cnn import build_activation_layer, build_norm_layer, xavier_init
 from mmcv.cnn.bricks.registry import (TRANSFORMER_LAYER,
                                       TRANSFORMER_LAYER_SEQUENCE)
 from mmcv.cnn.bricks.transformer import (BaseTransformerLayer,
+                                         MultiScaleDeformableAttention,
                                          TransformerLayerSequence,
                                          build_transformer_layer_sequence)
 from mmcv.runner.base_module import BaseModule
+from torch.nn.init import normal_
 
 from mmdet.models.utils.builder import TRANSFORMER
 
 
+def inverse_sigmoid(x, eps=1e-5):
```

Comment:
Add docstring, and import it in utils/__init__.py thus we can use it by `import inverse_sigmoid from mmdet.models.utils`.

Reply:
this would cause circular dependency.